tests/integration/app.test.ts

```diff
@@ -3,7 +3,7 @@ import request from "supertest";
 
 describe("Integration: Healthcheck", () => {
   test("Healthcheck endpoint returns 200", async () => {
-    const app = await createApp();
+    const app = createApp();
```

Comment:
Why is this needed, just out of curiosity?

Reply:
I think the test was setup as an example when the repo was created